cirq-superstaq/cirq_superstaq/daily_integration_test.py

```
@@ -177,6 +177,16 @@ def test_get_targets(service: css.Service) -> None:
     assert ibmq_target_info in result
     assert aqt_target_info in result
     assert all(target in result for target in filtered_result)
+    for gss_target in result:
+        target_name = gss_target.target
+        try:
+            assert (
+                service.target_info(target_name)["target"] == target_name
+            )  # Checks `.target_info()` is retrievable
+        except Exception as e:
```

why do we need to catch these? this is usually run automatically so we probably won't notice the prints if the test passes

---

My thought was the integration test failing and making an issue every single time a provider API fails might be too much, but having a debug check to review could be helpful. However, if this is rarer than I'd initially imagined I'm also happy to let the test fail normally